pkg/playground/test.js

```diff
@@ -130,6 +130,34 @@ document.addEventListener("DOMContentLoaded", () => {
         document.getElementById("user-info").textContent = JSON.stringify(info);
     });
 
+    const fsreplace_btn = document.getElementById("fsreplace1-create");
+    const fsreplace_error = document.getElementById("fsreplace1-error");
+    fsreplace_btn.addEventListener("click", e => {
+        fsreplace_btn.disabled = true;
+        fsreplace_error.textContent = '';
+        const filename = document.getElementById("fsreplace1-filename").value;
+        const content = document.getElementById("fsreplace1-content").value;
+        const attrs = { };
+        for (const field of ["owner", "group", "uid", "gid"]) {
+            let val = document.getElementById(`fsreplace1-${field}`).value;
+            if (!val)
+                continue;
+
+            if (field.endsWith('id'))
+                val = Number.parseInt(val, 10);
+
+            attrs[field] = val;
+        }
+        cockpit.file(filename, { superuser: "try" }).replace(content, undefined, attrs).then(() => {
```

Comment:
Please break the .then() line, to be on the same column as .catch". Or use `await`.